### PR TITLE
Update GOVERNANCE.md

### PR DESCRIPTION
**Pull request by canyon289**
cc @fonnesbeck @OriolAbril @junpenglao @twiecki @purna135 

----
:books: Documentation preview :books:: https://pymc--7031.org.readthedocs.build/en/7031/

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -191,6 +191,7 @@ Contributors don't need to be part of any dedicated team.
 * Michael Osthege (dev)
 * Oriol Abril-Pla (docs, community)
 * Osvaldo Martin (dev, docs)
+* Purna Chandra Mansingh (community(
```
Comment (purna135) [inline GOVERNANCE.md]:
```suggestion
* Purna Chandra Mansingh (community)
```

**Review by twiecki — APPROVED** [review]

**Comment by twiecki** [comment]
Can we also add @juanitorduz?

**Comment by OriolAbril** [comment]
We should double check that the list has everyone on https://github.com/orgs/pymc-devs/teams/core-contributors and viceversa

**Comment by canyon289** [comment]
Maybe we just should link to the github page and say that's that, unless we really care about. the labels. Definitely should add Juan but not sure what categories so Ill merge this one and whoever knows the categories can add in a followup

**Pull request merged**